pkg/chrootarchive/archive.go

```diff
@@ -35,13 +47,25 @@ func Untar(tarArchive io.Reader, dest string, options *archive.TarOptions) error
 		}
 	}
 
-	decompressedArchive, err := archive.DecompressStream(tarArchive)
-	if err != nil {
-		return err
+	var r io.ReadCloser = &noopCloser{Reader: tarArchive}
```

Comment:
i'll look at the ioutil implementation.

I looked into the invokeUnpack. Even though it doesn't close it, since it's used as cmd.Stdin, which is a File, I wasn't up for ensuring that just an `io.Reader` was sufficient.